guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc

```diff
@@ -0,0 +1,83 @@
+[id="Exporting_a_Content_View_Version_Syncable_{context}"]
+= Exporting a Content View Version in a Syncable Format
+
+You can export a version of a Content View to an archive file from {ProjectServer} in a syncable format.
```

Comment:
Why? When do I have to use the syncable format?
I wonder if we should describe the bigger picture of this scenario somewhere in the introduction.

Reply:
Maybe we could elaborate on how the content is syncable after it's exported?
Also, this kind of export does not result in an "archive file" :)

So perhaps something like: "You can export a version... to a format that can be used to synchronize the content from a local custom CDN?"